### PR TITLE
Feature/dali 45

### PR DESCRIPTION
**Pull request by noserronnie**
Create Posts with images

### CODE REVIEW

**Comment by Sashimi4** [comment]
I skipped the test due to that wrong detection

File: frontend/src/app/services/post/post.service.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export class PostService {
     return this.http.get<Post>(environment.apiUrl + 'posts/' + id);
   }
 
-  public createPost(post: Post): Observable<Post> {
+  public createPost(post: any): Observable<Post> {
```
Comment (davidemarcoli) [inline frontend/src/app/services/post/post.service.ts:23]: Please use the correct Type and not any

**Review by lazarpe — COMMENTED** [review]
Once this [comment](https://github.com/davidemarcoli/social-media-app/pull/36/files#r1090379305) is resolved. I'm happy to approve the changelist :)

**Review by Sashimi4 — APPROVED** [review]
LGTM

**Pull request merged**